homeassistant/components/insteon/manifest.json

```diff
@@ -15,7 +15,7 @@
   ],
   "documentation": "https://www.home-assistant.io/integrations/insteon",
   "iot_class": "local_push",
-  "loggers": ["pyinsteon", "pypubsub"],
+  "loggers": ["pyinsteon.messages", "pyinsteon.topics"],
```

Comment:
We really should include all places a message can get logged. I think that we shouldn't optimize this for the best debug experience, it should be set to include all possible logged messages.